lib/rules/no-extra-semicolons/__tests__/index.js

```diff
@@ -63,620 +64,779 @@ testRule(rule, {
   reject: [
     {
       code: ";",
+      fixed: "",
```

Comment:
Shouldn't the tests in this `config: [true],` section still include a single semi-colon?

Reply:
Are you saying it is better not to delete single semicolons?
if so, I think that this should be deleted because it is an extra semicolon.

Reply:
> if so, I think that this should be deleted because it is an extra semicolon.

Agreed. I think this semicolon is redundant and should be removed when autofixed.